postqa/jsonshim.py

```diff
@@ -15,18 +15,15 @@ def shim_validate_drp(vdrp_json, accepted_metrics):
 
     This makes a full job document, but only populates the ``measurements``
     field.
-
-    .. note::
-
-       This function does not currently validate the JSON it processes,
-       either as input or output.
     """
     job_json = {}
 
     measurements = []
     for vdrp_measurement_doc in vdrp_json['measurements']:
         if vdrp_measurement_doc['metric']['name'] not in accepted_metrics:
             continue
+        if vdrp_measurement_doc['value'] is None:
+            continue
```

Comment:
I think it make sense to skip metrics with None value. For a given metric if we have consecutive jobs with missing measurements this is highlighted in the time series plot and might indicate that 1) validade_drp run failed or 2) the dataset is not suitable to compute a specific metric.  I think we are adding 2) now.